scripts/docker/entrypoint_prod.sh

```diff
@@ -337,4 +337,10 @@ if [[ ${AIRFLOW_COMMAND} =~ ^(scheduler|celery)$ ]] \
     wait_for_celery_broker
 fi
 
+# Exit cleanly if the init was only intended to migrate DB
```

Comment:
Nit. Can you please also find a good place in docker compose docs or prod image docs to describe this as a "feature" ? It would be great to document it rather than people figuring it out by looking at the code.

Reply:
Nice! Thank you !